airflow/models/dag.py

```diff
@@ -2498,12 +2498,8 @@ def bulk_write_to_db(cls, dags: Collection["DAG"], session=NEW_SESSION):
             orm_dag.max_active_tasks = dag.max_active_tasks
             orm_dag.max_active_runs = dag.max_active_runs
             orm_dag.has_task_concurrency_limits = any(t.max_active_tis_per_dag is not None for t in dag.tasks)
-            if dag.schedule_on:
-                orm_dag.schedule_interval = 'Dataset'
-                orm_dag.timetable_description = 'Triggered by datasets.'
-            else:
-                orm_dag.schedule_interval = dag.schedule_interval
-                orm_dag.timetable_description = dag.timetable.description
+            orm_dag.schedule_interval = dag.schedule_interval
```

Comment:
i tried making it dag.timetable.summary --- which would be probably the "right" way but then tests started failing so i aborted mission

Reply:
Hmm those two should be the same value (due to the assignment in `__init__` so something somewhere is changing the value and we need to find out what that is.

Reply:
oh -- maybe i have that wrong! lemme look

Reply:
Yes the key “problem” per-se is that `DAG.schedule_interval` at this point does not actually have any functionality beyond determining the label name we show on the UI. A rename here is really needed to reflect that, and this change makes sense if you look pass the attribute name.

Reply:
yeah i guess what i am saying though is that while DagModel.schedule_interval does not have meaning and is just a label, DAG.schedule_interval _does_, and so that's why it feels a little off to store the label `Dataset` on `DAG.schedule_interval`.  storing on the DagModel attr makes sense but, in a perfect world, we wouldn't also set it on DagModel.  anyway, nothing really to worry about.

Reply:
> while DagModel.schedule_interval does not have meaning and is just a label, DAG.schedule_interval does

er, at least i think it does, doesn't it?  like that's where you set `@daily` or `30 7 * * *`